### DISCUSSION TITLE
failed using apt-get or apk

### DISCUSSION THREAD

**Discussion by ja114839** [ibm-messaging/mq-container#552] [General]
may I know the default package manager and the location of the package manager ?
I could not find it in the image and failed to run apt-get or apk update

**Comment by arthurbarr** [comment]
You can see in the [Dockerfile-server](https://github.com/ibm-messaging/mq-container/blob/b5d1875232ad45e321eb321e6fd4ebe429ab25ed/Dockerfile-server#L15), that the base image is `ibi-minimal`.  This is a Red Hat Universal Base Image, so uses the `dnf` package manager.  The minimal image which is used by MQ uses the `microdnf` package manager.  See [Red Hat Universal Base Image FAQ](https://developers.redhat.com/articles/ubi-faq) for more details. It should be possible to use an Ubuntu base image instead, though we don't routinely test the sample with that.
